sections/04-open-source.qmd

```diff
@@ -460,4 +458,110 @@ r |>
   ggtitle("R Libs")
 ```
 
-::: 
\ No newline at end of file
+::: 
+
+::: {.page-banner image_paths='("assets/images/hexes/RStudio.svg",)'}
+
+## RStudio Integrated Development Environment
+
+::: {.cols-2}
+
+[RStudio](https://posit.co/products/open-source/rstudio/)
+is a multi-language IDE designed for Data Science with R and Python. It
+augments the standard code console with an editor that can display
+Notebooks, launch apps, highlight code syntax, spot code errors, and
+directly execute code. Built into the IDE are tools for debugging,
+plotting, browsing files, and managing project histories and workspaces.
+Together these tools make data scientists and developers much more
+efficient.
+
+{{< colbreak >}} 
+
+There are 5 FTE Posit employees developing the RStudio IDE open-source
+desktop and server products as of May 2024.
+
+:::
+
+```{r}
+#| label: rstudio
+rstudio |> growth_plot() + ggtitle("RStudio")
+```
+
+:::
+
+# Additional Posit Free-to-Use Solutions
```

Comment:
This level 1 heading generates a new blank page in the PDF version of the report:
<img width="679" alt="Screenshot 2025-05-09 at 10 25 23 AM" src="https://github.com/user-attachments/assets/1c46fa17-28ff-4914-b995-f82ff8a03d1c" />
Some options:
* Add an introduction to this section to make the page less blank.
* Move the heading to the Positron page, although that may make it less clear that "Posit Package Manager" is also part of this section.
* Put the  "free-to-use" items in the "Open source" section, but explain the distinction in the introduction, then add either "Open-source" or "Free-to-Use" on the left hand side of the page banners.